checkov/terraform/checks/resource/azure/ACREnableZoneRedundancy.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+from typing import Any
+
+
+class ACREnableZoneRedundancy(BaseResourceCheck):
+
+    def __init__(self) -> None:
+        """
+        Zone redundancy provides resiliency and high availability to
+        a registry or replication resource in a specific region.
+        """
+        name = "Ensure Azure Container Registry (ACR) is zone redundant"
+        id = "CKV_AZURE_233"
+        supported_resources = ("azurerm_container_registry",)
+        categories = (CheckCategories.BACKUP_AND_RECOVERY,)
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf: dict[str, list[Any]]) -> CheckResult:
+        # supported only with Premium
+        sku = conf.get("sku")
+        if sku != ["Premium"]:
+            return CheckResult.PASSED
```

Comment:
can you remove this block, because the point of the check is to enable zone redundancy and if it means to actually set it to `Premium` tier so be it. And don't forget to adjust the related test resource from passing to failing, thansk!

Reply:
done